kitchen.dokken.yml

```diff
@@ -0,0 +1,22 @@
+---
+driver:
+  name: dokken
+  privileged: true
+
+transport:
+  name: dokken
+
+provisioner:
+  name: dokken
+  chef_version: <%= ENV['CHEF_VERSION'] || 'current' %>
+  chef_license: accept-no-persist
```

Comment:
These actually needs to be moved into the `driver` section. For whatever reason, this is how the dokken plugin works.